--- common.h	2015-09-17 19:43:23.260749824 +0200
+++ common.new.h	2015-09-17 19:46:25.583886880 +0200
@@ -28,14 +28,14 @@
     ZMARGIN_WALL = 6,
 };
 
-const int numberdensity = 4;
-const float dt = 0.001;
-const float kBT = 0.0945;
-const float gammadpd = 45;
+const int numberdensity = 4; //=const int numberdensity = %numberdensity%;
+const float dt = 0.001; //=const float dt = %dt%;
+const float kBT = 0.0945; //=const float kBT = %kBT%;
+const float gammadpd = 45; //=const float gammadpd = %gammadpd%;
 const float sigma = sqrt(2 * gammadpd * kBT);
 const float sigmaf = sigma / sqrt(dt);
-const float aij = 25;
-const float hydrostatic_a = 0.05;
+const float aij = 25; //=const float aij = %aij%;
+const float hydrostatic_a = 0.05; //=const float hydrostatic_a = %hydrostatica%;
 
 extern float tend;
 extern bool walls, pushtheflow, doublepoiseuille, rbcs, ctcs, xyz_dumps, hdf5field_dumps, hdf5part_dumps, is_mps_enabled;
